boa/src/builtins/value/display.rs

```diff
@@ -124,14 +111,10 @@ pub(crate) fn log_string_from(x: &ValueData, print_internals: bool) -> String {
                 _ => display_obj(&x, print_internals),
             }
         }
-        ValueData::Symbol(ref sym) => {
-            let desc: Value = sym.borrow().get_internal_slot("Description");
-            match *desc {
-                ValueData::String(ref st) => format!("Symbol(\"{}\")", st.to_string()),
-                _ => String::from("Symbol()"),
-            }
-        }
-
+        ValueData::Symbol(ref symbol) => match symbol.description() {
+            Some(description) => format!("Symbol({})", description),
+            None => String::from("Symbol()"),
+        },
```

Comment:
Do you think that this would be easier to understand?
```rust
symbol.description.map(|desc| format!("Symbol({})", desc)).unwrap_or_else(||String::from("Symbol()"))
```

Reply:
```rust
symbol.description.map_or_else(|| String::from("Symbol()"), |desc| format!("Symbol({})", desc))
```
I think with `map_or_else` its much better, right?

Reply:
It should s, yes :)